packages/forms-web-app/src/constants.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  constants: {
+    eligibility: {
+      planningPermissionStatus: {
+        GRANTED: 'granted',
```

Comment:
For consistency for the params it would be better to use either uppercase or camel case but not both.